libs/libtins/Makefile

```diff
@@ -49,6 +49,7 @@ endef
 
 define Package/libtins/install
 	$(INSTALL_DIR) $(1)/usr/lib
+	$(INSTALL_DATA) $(PKG_INSTALL_DIR)/usr/lib/libtins.so $(1)/usr/lib/
```

Comment:
why not just fix the next line to be libtins.so* instead of .*?

Reply:
Because that would result in a link loop. I tried it first